spec/tasks/rake_spec.rb

```diff
@@ -1,12 +1,15 @@
 require 'rake'
 
 RSpec.describe "Rake tasks" do
+  before(:all) do
+    Rails.application.load_tasks
```

Comment:
Using this idiomatic call in favor of our custom `load_rake_environment` method, which I have excised above.